packages/popover/src/vaadin-popover-overlay.js

```diff
@@ -62,8 +62,7 @@ class PopoverOverlay extends PopoverOverlayMixin(DirMixin(ThemableMixin(PolylitM
           content: '';
           top: calc(var(--vaadin-popover-offset-top, 0) * -1);
           bottom: calc(var(--vaadin-popover-offset-bottom, 0) * -1);
-          inset-inline-start: calc(var(--vaadin-popover-offset-start, 0) * -1);
-          inset-inline-end: calc(var(--vaadin-popover-offset-end, 0) * -1);
+          inset-inline: calc(var(--vaadin-popover-offset-start, 0) * -1);
```

Comment:
This change isn't equivalent. It now uses `--vaadin-popover-offset-start` for both start and end inset-inline.

Reply:
My bad, fixed.